example/ultravox_example/client.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import asyncio
+import logging
+import signal
+import sys
+
+import ultravox_client as uv
+
+
+async def main():
+    session = uv.UltravoxSession(experimental_messages=bool(args.experimental_messages))
+    state = await session.joinCall(args.join_url)
```

Comment:
Because I might be better a translating code than an LLM, but I'm also more likely to forget to translate cases. :sweat_smile: 

Thanks for catching that fixed.